tests/unit/router.js

```diff
@@ -499,6 +499,22 @@ define([
       });
     });
 
+    describe('bic/router/middleware/login', function () {
+      var middleware;
+
+      beforeEach(function (done) {
+        injector.require(['bic/router/middleware/login'], function (mw) {
+          middleware = mw;
+          done();
+        });
+      });
+
+      it('exports a Function, with 3 arguments', function () {
+        assert.isFunction(middleware);
+        assert.lengthOf(middleware, 3);
+      });
+    });
+
```

Comment:
is it possible/worth it to use sinon to stub out external functions and confirm the function works as intended?